src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/PostSetup.java

```diff
@@ -109,6 +110,7 @@ public static void loadItems() {
 
         Slimefun.getItemCfg().save();
         Slimefun.getResearchCfg().save();
+        Bukkit.getPluginManager().callEvent(new SlimefunRegistryFinalizedEvent());
```

Comment:
Seems like a bit of a strange place to put this call, any reason it's not higher up? Either right under item load or right under the recipe load.